Formula/libpulsar.rb

```diff
@@ -0,0 +1,44 @@
+class Libpulsar < Formula
+  desc "Apache Pulsar C++ library"
+  homepage "https://pulsar.apache.org"
+
+  url "https://www.apache.org/dyn/mirrors/mirrors.cgi?action=download&filename=pulsar/pulsar-2.2.0/apache-pulsar-2.2.0-src.tar.gz"
+  sha256 "a3b1940a803043bb2c365ce9df657d15bf9aacb3c9ff5db86a79dc4374033f08"
+
+  depends_on "boost-python" => :build
+  depends_on "cmake" => :build
+  depends_on "jsoncpp" => :build
+  depends_on "openssl" => :build
+  depends_on "pkg-config" => :build
+  depends_on "protobuf@2.6" => :build
```

Comment:
Hum… doesn't it build against the latest protobuf? We normal don't like depending on older versions.

Reply:
Good point. Fixed to use standard protobuf.